Assets/Scripts/PlayerController.cs

```diff
@@ -21,28 +41,31 @@ public class PlayerController : MonoBehaviour {
     [SerializeField]
     private ContactFilter2D groundContactFilter;
     [SerializeField]
-    private Text collectableText;
-    [SerializeField]
     private Slider overheatSlider;
```

Comment:
Wrap all these serialized fields up in a #region https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives/preprocessor-region